includes/core-changes/codeanalysis/5.0/ca1416-platform-compatibility-analyzer.md

```diff
@@ -1,14 +1,14 @@
 ### CA1416: Platform compatibility
 
-.NET code analyzer rule CA1416 is enabled, by default, starting in .NET 5.0. It produces a build warning for calls to platform-specific APIs from call sites that don't verify the operating system.
+.NET code analyzer rule [CA1416](/visualstudio/code-quality/ca1416) is enabled, by default, starting in .NET 5.0. It produces a build warning for calls to platform-specific APIs from call sites that don't verify the operating system.
```

Comment:
Isn't it need the extension? 
```suggestion
.NET code analyzer rule [CA1416](/visualstudio/code-quality/ca1416.md) is enabled, by default, starting in .NET 5.0. It produces a build warning for calls to platform-specific APIs from call sites that don't verify the operating system.
```

Reply:
Since this is a "site-relative" link (i.e. it starts with a / character), it doesn't need the .md extension. In fact, including it would break the link. You only use the file extension for relative links to files in the same repo/docset.

Reply:
Good to know, thanks